x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.test.tsx

```diff
@@ -48,15 +64,28 @@ describe('CrawlRequestsTable', () => {
 
   describe('columns', () => {
     beforeAll(() => {
+      setMockActions(actions);
       setMockValues(values);
       wrapper = shallow(<CrawlRequestsTable />);
       tableContent = mountWithIntl(<CrawlRequestsTable />)
         .find(EuiBasicTable)
         .text();
     });
 
-    it('renders an id column', () => {
+    it('renders a id column ', () => {
+      expect(tableContent).toContain('Request ID');
       expect(tableContent).toContain('618d0e66abe97bc688328900');
+      expect(tableContent).toContain('54325423aef7890543');
+
+      // TODO Use data-test-subj instead, currently not able to find inside the table
+      // expect(wrapper.find('[data-test-subj="CrawlRequestsTableId"]').children()).toContain(
+      //   '54325423aef7890543'
+      // );
+      // expect(wrapper.find('[data-test-subj="CrawlRequestsTableIdLink"]').children()).toContain(
+      //   '618d0e66abe97bc688328900'
+      // );
+      // wrapper.find('[data-test-subj="CrawlRequestsTableIdLink"]').simulate('click');
+      // expect(actions.fetchCrawlRequest).toHaveBeenCalledWith('618d0e66abe97bc688328900');
```

Comment:
I could use an assist with this to be honest

Reply:
This is an approach I've taken elsewhere, to get a rendered column from a shallow rendered table.

https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/suggestions_table.test.tsx#L56-L83

Does this help?

Reply:
Yes! I've taken this approach in 245d6f6 thanks for sending it my way